src/Products/Signature/Controllers/SignatureApiController.cs

```diff
@@ -1124,21 +1141,20 @@ private SignedDocumentEntity SignDocument(string documentGuid, string password,
         /// <param name="password">string</param>
         /// <param name="signsCollection">SignatureOptionsCollection</param>
         /// <returns></returns>
-        private Stream SignDocumentStream(string documentGuid, string password, SignatureOptionsCollection signsCollection)
+        private Stream SignDocumentStream(string documentGuid, string password, List<SignOptions> signsCollection)
         {
             // set save options
             SaveOptions saveOptions = new SaveOptions();
-            saveOptions.OutputType = OutputType.Stream;
-            saveOptions.OutputFileName = Path.GetFileName(documentGuid);
 
-            // set password
-            LoadOptions loadOptions = new LoadOptions();
-            if (!String.IsNullOrEmpty(password))
+            using (FileStream fileStream = File.Open(documentGuid, FileMode.Open, FileAccess.ReadWrite, FileShare.ReadWrite))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the 'using' statement; it will cause automatic disposal of 'fileStream'.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Signature-for-.NET-MVC/pullRequest?prid=5325147)